packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade/index.scss

```diff
@@ -0,0 +1,3 @@
+.upgrade-intervention-calender .flatpickr-day.today.flatpickr-disabled {
+  color: rgba(57, 57, 57, 0.1);
```

Comment:
Can't we use an existing color from ui kit ?

Reply:
I need to set color same as other disabled dates of the calendar component. I will check if the design has the predefined color.

Reply:
I did not find the same color

Reply:
Should be done in UIKit ?

Reply:
May be yes, its a good requirement for uikit. If the current date is disabled, applay different style so that it does not appear like enabled.

Reply:
Created a bug on ui-kit